### PR TITLE
Arch: Add aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by ax3l**
Add builds for these new platforms.

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
Hi @jschuller,
just adding support for ppc64le (and ARM) because we want to use `nlopt` on [Summit](https://www.olcf.ornl.gov/summit/) :)

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
Hi @jschueller,
just adding support for ppc64le (and ARM) because we want to use `nlopt` on [Summit](https://www.olcf.ornl.gov/summit/) :)

**Comment by isuruf** [comment]
Can you fix https://github.com/conda-forge/nlopt-feedstock/pull/25#issuecomment-610446385 ?

**Comment by ax3l** [comment]
@isuruf already fixed, linter passes now (but for some reason did not comment again).

**Comment by ax3l** [comment]
oh wait, right still off

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +34,7 @@ test:
 
 about:
   home: http://ab-initio.mit.edu/nlopt
-  license: LGPL 2.1
+  license: LGPL-2.1
```
Comment (isuruf) [inline recipe/meta.yaml]: It's either `LGPL-2.1-only` or `LGPL-2.1-or-later`. You'll have to check their docs

Comment (ax3l) [reply recipe/meta.yaml]: just saw, sorry. It's `-or-later`: https://github.com/stevengj/nlopt/blob/master/COPYING

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@isuruf now we are ready, SPDX identified fixed :)

**Comment by isuruf** [comment]
Do you want to be a maintainer for nlopt?

**Comment by ax3l** [comment]
Thanks but not necessarily, just quickly providing this update for collaborators (on which we depend downstream).

**Comment by isuruf** [comment]
Thanks @ax3l. Excited to see conda-forge being used on Summit.

**Comment by ax3l** [comment]
Thanks to your excellent `ppc64le` templates, I am happily updating our software stack since a year or so :)

@jschueller thanks for the merge! :)

**Pull request merged**